storybookjs/storybook
# UI: Scroll to highlighted search result

Issue: https://github.com/storybookjs/storybook/issues/21323

## What I did

- Scroll the highlighted search result into view when using keyboard to navigate between search results

## How to test

- Start any storybook sandbox template.
- Enter "e" in search input to have enough search results, using keyboard down to navigate between results to see the focused result is always in view.

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.
